google-cloud-clients/google-cloud-compute/synth.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import os
+import synthtool as s
+import logging
+
+from synthtool import log
+from synthtool import _tracked_paths
+from synthtool import shell
+from synthtool.sources import git
+
+logging.basicConfig(level=logging.DEBUG)
+
+discovery_url = "https://github.com/googleapis/discovery-artifact-manager.git"
+
+log.debug(f"Cloning {discovery_url}.")
+discovery = git.clone(discovery_url, depth=1)
+
+def generate(config, root_dir):
```

Comment:
Let's just go ahead and get this upstreamed.

Reply:
Ok, I'll remove from here and get it into synthtool